openslides/assignments/static/templates/assignments/assignment-detail.html

```diff
@@ -53,6 +53,13 @@ <h3 translate>Number of posts to be elected</h3>
           {{ assignment.open_posts }}<br>
       </div>
       <div class="col-md-6">
```

Comment:
@joelmacht Please use a 2-column layout (2 times col-md-6) and move the tags below the 'posts'. After your change I'll test it.